resources/lang/en/admin/depreciations/general.php

```diff
@@ -8,5 +8,5 @@
     'depreciation_name'  					=> 'Depreciation Name',
     'number_of_months'  					=> 'Number of Months',
     'update'  					            => 'Update Depreciation',
-
+    'min_value'                             => 'Set Minimum Value after Depreciation',
```

Comment:
I'd maybe change this to something a little clearer, like "Min value after fully depreciated" and then add `min_value_help` which explains this a little further. It might be confusing as is. (That's not your fault, different companies depreciate differently, so not everyone will know what this means.)